playbooks/hadoop-build-arm64/run.yaml

```diff
@@ -0,0 +1,32 @@
+- hosts: all
+  become: yes
+  roles:
+    - install-openjdk
+    - install-maven
+  tasks:
+    - name: install required packages
```

Comment:
This patch contain two step workaround, please add comments for:
1. apply protobuf 2.5.0 aarch64 enabled patch, recompile and install
2. copy protoc-gen-grpc-java 1.15.1 aarch enabled package into local Mave repository